code/modules/mob/living/carbon/carbon.dm

```diff
@@ -812,3 +812,17 @@ var/const/GALOSHES_DONT_HELP = 4
 	. = ..()
 	if(!getorgan(/obj/item/organ/internal/brain))
 		return 0
+
+/mob/living/carbon/harvest(mob/living/user)
+	if(qdeleted(src))
+		return
+	var/organs_amt = 0
+	for(var/obj/item/organ/internal/O in internal_organs)

```

Comment:
1. @KazeEspada ithe term is istypeless for loop.
2. internal_organs list DOES contain non-organs, so it's not safe to use.